data/changelog.txt

```diff
@@ -11,13 +13,25 @@ Add generic finales for generic mine
 Add new boss for physics lab
 Add 4 New Apartment Complexes
 Add several new bungalows
+Sugar house expansion
+Add Bungalows 21 and 22
+Various additions and tweaks relating to Native American Heritage
+Add a drug lab
+Environmental storytelling about what children in the cataclysm had to go through
+Finish part 2 of the netherium monster corpse
```

Comment:
```suggestion
Add Netherium monster corpse map location
```

Reply:
I'd delete this one since it's being reverted until it stops breaking tests.